cypress/plugins/index.js

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
```

Comment:
The reference comment is not required, as Cypress automatically includes its types.